packages/token-search-discovery-controller/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@metamask/token-search-discovery-controller",
+  "version": "0.1.0",
```

Comment:
We should not start this off at 0.1.0 because if this gets included in a release it will get published accidentally. Let's start this at 0.0.0 and then we can bump it in a release:

``` suggestion
  "version": "0.0.0",